Docker build fails due to proxy error

### **EXPECTED RESULTS**
Build succeeds

### **ACTUAL RESULTS**
Build fails with error:
```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.665 s
[INFO] Finished at: 2019-04-23T10:51:23Z
[INFO] ------------------------------------------------------------------------
[ERROR] Plugin org.apache.maven.plugins:maven-checkstyle-plugin:2.11 or one of its dependencies could not be resolved: Failed to read artifact descriptor for org.apache.maven.plugins:maven-checkstyle-plugin:jar:2.11: Could not transfer artifact org.apache.maven.plugins:maven-checkstyle-plugin:pom:2.11 from/to central (https://repo.maven.apache.org/maven2): repo.maven.apache.org: Name or service not known: Unknown host repo.maven.apache.org: Name or service not known -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginResolutionException
The command '/bin/sh -c mvn --batch-mode --define java.net.useSystemProxies=true package' returned a non-zero code: 1
```
### **STEPS TO REPRODUCE**
```
$ git clone https://github.com/plantuml/plantuml-server.git
$ cd plantuml-server/
$ docker build --build-arg http_proxy=http://myproxy/ --build-arg https_proxy=http://myproxy/ -t plantuml:test -f Dockerfile.jetty .
```
### **ERROR LOG**
```
$ git clone https://github.com/plantuml/plantuml-server.git
Cloning into 'plantuml-server'...
remote: Enumerating objects: 5, done.
remote: Counting objects: 100% (5/5), done.
remote: Compressing objects: 100% (5/5), done.
remote: Total 1939 (delta 0), reused 3 (delta 0), pack-reused 1934
Receiving objects: 100% (1939/1939), 6.71 MiB | 635.00 KiB/s, done.
Resolving deltas: 100% (706/706), done.
Checking connectivity... done.

$ cd plantuml-server/

$ docker build --build-arg http_proxy=http://myproxy/ --build-arg https_proxy=http://myproxy/ -t plantuml:test -f Dockerfile.jetty .
Sending build context to Docker daemon  259.1kB
Step 1/13 : FROM maven:3.6-jdk-8-slim AS builderjetty
3.6-jdk-8-slim: Pulling from library/maven
27833a3ba0a5: Already exists
16d944e3d00d: Pull complete
9019de9fce5f: Pull complete
9b053055f644: Pull complete
1c80aca6b8ec: Pull complete
38946562d2a7: Pull complete
96b0df30f304: Pull complete
ad67fddf8221: Pull complete
44a05031bddb: Pull complete
Digest: sha256:dce33cc7a4702cc5f3ea3a6deb4ea840c17001895ffe169d96e1fd9d7041eb15
Status: Downloaded newer image for maven:3.6-jdk-8-slim
 ---> b92711a7869c
Step 2/13 : COPY pom.xml /app/
 ---> 08810654b6a2
Step 3/13 : COPY src /app/src/
 ---> ae5378c98976
Step 4/13 : WORKDIR /app
 ---> Running in ca49caa9b409
Removing intermediate container ca49caa9b409
 ---> 68bb66250e6b
Step 5/13 : RUN mvn --batch-mode --define java.net.useSystemProxies=true package
 ---> Running in 929eedbc091f
[INFO] Scanning for projects...
[INFO]
[INFO] --------------< org.sourceforge.plantuml:plantumlservlet >--------------
[INFO] Building PlantUML Servlet 1-SNAPSHOT
[INFO] --------------------------------[ war ]---------------------------------
[INFO] Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-checkstyle-plugin/2.11/maven-checkstyle-plugin-2.11.pom
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.665 s
[INFO] Finished at: 2019-04-23T10:51:23Z
[INFO] ------------------------------------------------------------------------
[ERROR] Plugin org.apache.maven.plugins:maven-checkstyle-plugin:2.11 or one of its dependencies could not be resolved: Failed to read artifact descriptor for org.apache.maven.plugins:maven-checkstyle-plugin:jar:2.11: Could not transfer artifact org.apache.maven.plugins:maven-checkstyle-plugin:pom:2.11 from/to central (https://repo.maven.apache.org/maven2): repo.maven.apache.org: Name or service not known: Unknown host repo.maven.apache.org: Name or service not known -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginResolutionException
The command '/bin/sh -c mvn --batch-mode --define java.net.useSystemProxies=true package' returned a non-zero code: 1
```

--- a/Dockerfile.jetty
+++ b/Dockerfile.jetty
@@ -1,4 +1,4 @@
-FROM maven:3.6-jdk-8-slim AS builderjetty
+FROM maven:3.6-jdk-8 AS builderjetty
 
 COPY pom.xml /app/
 COPY src /app/src/

--- a/Dockerfile.tomcat
+++ b/Dockerfile.tomcat
@@ -1,4 +1,4 @@
-FROM maven:3.6-jdk-8-slim AS buildertomcat
+FROM maven:3.6-jdk-8 AS buildertomcat
 
 COPY pom.xml /app/
 COPY src /app/src/

